src/tools/MappingTools.ts

```diff
@@ -41,32 +42,72 @@ export function mapFile(sourceFile: ts.SourceFile) {
           break;
         case ts.SyntaxKind.InterfaceDeclaration:
           file.addInterface(
-            mapInterface(<ts.InterfaceDeclaration>child, sourceFile),
+            mapInterface(<ts.InterfaceDeclaration>child, sourceFile)
           );
           break;
         case ts.SyntaxKind.VariableStatement:
           file.addVariable(
-            mapVariableStatement(<ts.VariableStatement>child, sourceFile),
+            mapVariableStatement(<ts.VariableStatement>child, sourceFile)
           );
           break;
         case ts.SyntaxKind.FunctionDeclaration:
           file.addFunction(
-            mapFunction(<ts.FunctionDeclaration>child, sourceFile),
+            mapFunction(<ts.FunctionDeclaration>child, sourceFile)
           );
           break;
         case ts.SyntaxKind.EnumDeclaration:
           file.addEnum(mapEnums(<ts.EnumDeclaration>child));
           break;
         case ts.SyntaxKind.ExpressionStatement:
           file.addExpression(
-            mapExpressions(<ts.ExpressionStatement>child, sourceFile),
+            mapExpressions(<ts.ExpressionStatement>child, sourceFile)
           );
           break;
+        case ts.SyntaxKind.ExportKeyword:
+          let lineText: string[] = sourceFile.getText().split("\n");
```

Comment:
You can also get the line with child.getText()

Reply:
See comment above